flask_iiif/signals.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Flask-IIIF
+# Copyright (C) 2014, 2015 CERN.
+#
+# Flask-IIIF is free software; you can redistribute it and/or modify
+# it under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Flask IIIF Signals."""
+
+from blinker import Namespace
+
+_signals = Namespace()
+
+# Before request
+iiif_before_process_request = _signals.signal('iiif-before-process-request')
```

Comment:
Please document what data are sent in the signal.

Example: https://github.com/inveniosoftware/invenio/blob/master/invenio/modules/accounts/signals.py